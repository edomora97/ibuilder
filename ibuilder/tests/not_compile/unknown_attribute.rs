#[derive(ibuilder::ibuilder)]
struct Foo {
    #[ibuilder(wibble_monster = 42)]
    field: i64,
}

#[derive(ibuilder::ibuilder)]
#[ibuilder(wibble_monster = 42)]
struct Bar {
    field: i64,
}

#[derive(ibuilder::ibuilder)]
#[ibuilder(wibble_monster = 42)]
enum Baz {
    Var,
}

#[derive(ibuilder::ibuilder)]
enum Bim {
    #[ibuilder(wibble_monster = 42)]
    Var,
}

fn main() {}
