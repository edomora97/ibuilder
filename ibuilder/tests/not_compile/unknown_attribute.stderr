error: unknown attribute
 --> $DIR/unknown_attribute.rs:3:16
  |
3 |     #[ibuilder(wibble_monster = 42)]
  |                ^^^^^^^^^^^^^^

error: unknown attribute
 --> $DIR/unknown_attribute.rs:8:12
  |
8 | #[ibuilder(wibble_monster = 42)]
  |            ^^^^^^^^^^^^^^

error: unknown attribute
  --> $DIR/unknown_attribute.rs:14:12
   |
14 | #[ibuilder(wibble_monster = 42)]
   |            ^^^^^^^^^^^^^^

error: unknown attribute
  --> $DIR/unknown_attribute.rs:21:16
   |
21 |     #[ibuilder(wibble_monster = 42)]
   |                ^^^^^^^^^^^^^^
